### PR TITLE
Update service-roles-new-workspaces.md

### PR DESCRIPTION
**Pull request by UnkemptDataBod**
Clarified there are permissions that necessiate an admin enabling a feature.  Simplified the permissions table

### CODE REVIEW

File: powerbi-docs/collaborate-share/service-roles-new-workspaces.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 title: Roles in workspaces in Power BI
 description: Learn about the different roles you can assign to workspace users to grant access to read, write, edit, and more.
-author: maggiesMSFT
+author: UnkemptDataBod
```
Comment (Jak-MS) [inline powerbi-docs/collaborate-share/service-roles-new-workspaces.md:4]:
```suggestion
author: maggiesMSFT
```
authors must be Microsoft content team members

File: powerbi-docs/collaborate-share/service-roles-new-workspaces.md [hunk 2]
```diff
@@ -62,6 +61,10 @@ Everyone in a user group gets the role that you've assigned. If someone is in se
 
 <sup>5</sup> If the items are in a workspace in a Premium capacity, you can view and interact with items in the Power BI service even if you don't have a Power BI Pro license.
 
+<sup>6</sup> If your admin has enabled this feature, see [Manage featured content](https://learn.microsoft.com/fabric/admin/service-admin-portal-featured-content)
```
Comment (Jak-MS) [inline powerbi-docs/collaborate-share/service-roles-new-workspaces.md:64]:
```suggestion
<sup>6</sup> If your admin has enabled this feature, see [Manage featured content](/fabric/admin/service-admin-portal-featured-content)
```

File: powerbi-docs/collaborate-share/service-roles-new-workspaces.md [hunk 3]
```diff
@@ -62,6 +61,10 @@ Everyone in a user group gets the role that you've assigned. If someone is in se
...
+
+<sup>7</sup> If your admin has enabled this feature. see [Users can try Microsoft Fabric paid features](https://learn.microsoft.com/fabric/admin/service-admin-portal-help-support#users-can-try-microsoft-fabric-paid-features)
```
Comment (Jak-MS) [inline powerbi-docs/collaborate-share/service-roles-new-workspaces.md:66]:
```suggestion
<sup>7</sup> If your admin has enabled this feature. see [Users can try Microsoft Fabric paid features](/fabric/admin/service-admin-portal-help-support#users-can-try-microsoft-fabric-paid-features)
```

**Comment by Jak-MS** [comment]
@maggiesMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by UnkemptDataBod** [comment]
@maggiesMSFT 

I have updated the PR to account for changes made since raising, there are currently no conflicts and I believe the information I have updated is relevant and correct. Has this stalled somewhere?

**Comment by maggiesMSFT** [comment]
Hi, @UnkemptDataBod -- I've made a PR with your update, so we can view it in preview, which we can't do for updates in the public repo. So I'm closing your PR. I have sent my updated preview to the PMs to ask them to review before I sign off. Thanks!
Replacing with https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/11238

**Pull request closed without merging**